src/gui/Src/Gui/CPUDump.cpp

```diff
@@ -515,6 +515,30 @@ void CPUDump::mouseMoveEvent(QMouseEvent* event)
     HexDump::mouseMoveEvent(event);
 }
 
+void CPUDump::mousePressEvent(QMouseEvent *event)
+{
+    if(((event->buttons() & Qt::LeftButton) != 0) && ((event->buttons() & Qt::RightButton) == 0))
+    {
+        if(getColumnCount() && (event->y() <= getHeaderHeight()) &&event->y() >= 0 )
+        {
+            bool wHasCursor = cursor().shape() == Qt::SplitHCursor ? true : false;
+            if(!wHasCursor)
+            {
+                int wColIndex = getColumnIndexFromX(event->x());
+                if( wColIndex == 2)
+                {
+                    QString title = getColTitle(wColIndex);
+                    if(!title.compare("UNICODE"))

```

Comment:
This is not really a property you can depend on and it would only work on the English version...